src/models/__tests__/user.test.integration.ts

```diff
@@ -21,9 +21,9 @@ describe('User Model', () => {
 
   describe('Read', () => {
     test('get :: no arguments gets all users', async () => {
-      const users = await Users.get(knex);
+      const users = await Users.get(knex, { order: ['id', 'asc'] });
```

Comment:
This test case is for calling `get` with no arguments (other than the client), so I'd prefer the assertion to change instead of the query here.

The assertion can become something like:
```js
expect(users).toEqual(expect.arrayContaining(expect.objectContaining({ name: 'Chell', ... })));
```